src/server/status/plugin_status.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import states from './states';
+import Status from './status';
+
+class PluginStatus extends Status {
+  constructor(plugin, server) {
+    super(`plugin:${plugin.id}@${plugin.version}`, server);
+    this.plugin = plugin;
+  }
+
+  toJSON() {
```

Comment:
Do we need to override `toJSON()` at this time? AFAICT, nothing is actually using the properties specific to plugins except the unit tests. We could implement the overridden method at a later time if/when we actually need to expose certain specific plugin properties.